sdh-proxy.c

```diff
@@ -136,6 +145,36 @@ unsigned short int * udp_get_port( const short unsigned int pktlen, const u_char
 
 }
 
+/** 
+ * function to log some statistical Ouitput to a logfile
+ * This is usefull to get somewhat of information / statistical logging.
+ *
+
+ *
+ **/
+void writeLogStats()
+{
+    // don't spam! just write the log once every X seconds
+    if((logtimer + 10) < (int)time(NULL)) {
```

Comment:
I think this is missing the value being update after each loop, so this will run every time a packet is forwarded?